Example/Tests/TransformControllerSpec.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  TransformControllerTests.swift
+//  AirRivet
+//
+//  Created by Hans Van Herreweghe on 21/04/16.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import XCTest
+import Nimble
+import Quick
+@testable import AirRivet
+
+class ExampleBaseModel: UniqueAble, Mitigatable, Parsable {
+    var objectId: String?
+
+	required init (){
+
+	}
+	
+    func parseFromDict(json: AnyObject) throws {
+        if let json = json as? NSDictionary,
+            identifier = json["identifier"] as? String {
+                self.objectId = identifier
+		}else {
+			throw ResponseError.InvalidResponseData
+		}
+    }
+    
+    func toDictionary()-> NSDictionary? {
+        return [
+            "identifier": objectId!,
+        ]
+    }
+
+	static func rootKey() -> String? {
+		return "results"
+	}
+
+	func responseMitigator() -> ResponsMitigatable {
+		return DefaultMitigator()
+	}
+
+	static func requestMitigator()-> RequestMitigatable {
+		return DefaultMitigator()
+	}
+}
+extension ExampleBaseModel: EnvironmentConfigurable {
+
+	func contextPath() -> String {
+		return "something"
+	}
+	
+	func environment() -> protocol<Environment, Mockable, Transformable> {
+		return Parse<GameScore>() //TODO make this a mock
+	}
+}
+
+class TransformControllerSpec: QuickSpec {
+    
+	private func loadDataFromUrl(url: String) throws -> NSData? {
+		guard
+			let path = NSBundle.mainBundle().pathForResource(url, ofType: "json"),
+			let data = NSData(contentsOfFile: path) else {
+				XCTFail("problem loading json")
+				return nil
+		}
+		return data
+	}
+    //MARK: transform
+    
+    override func spec()
+    {
+        let transformController = TransformController()
+        var data = NSData()
+        describe("TransformController"){
+            
+            it("should not return error at loadingData"){
+                expect {try data = self.loadDataFromUrl("exampleBaseModel")!}.notTo(throwError())
+            }
+            
+            it ("should return correct objectId at transformation"){
+                try! transformController.transform(data, completion: {
+                    (model: ExampleBaseModel) in
+					expect(model.objectId).to(equal("123456ABCdef"))
```

Comment:
@HannesVDB Ik heb de expect binnen de closure gezet en de cast onnodig gemaakt. Nu moet je wel expliciet het verwachte type zetten bij de closure. Dat is het gevolg van de refactor van gisteren. Maar lijkt me mooier?